app/src/main/cpp/native-lib.cpp

```diff
@@ -3,27 +3,21 @@
 #include <bnb/error.h>
 #include <bnb/utility_manager.h>
 
-#include "offscreen_effect_player.hpp"
-#include "offscreen_render_target.hpp"
-#include "thread_pool.h"
+#include <iostream>
+
+#include "effect_player.hpp"
+#include "render_context.hpp"
+#include "oep/interfaces/pixel_buffer.hpp"
+#include "oep/interfaces/image_format.hpp"
+#include "oep/interfaces/offscreen_effect_player.hpp"
+#include "oep/interfaces/offscreen_render_target.hpp"
 
 #include <android/log.h>
 #include <jni.h>
-
```

Comment:
better to return this empty line, just to separate Android-specific and std headers